examples/simple_project/main.tf

```
@@ -28,10 +28,13 @@ locals {
   subnet_03 = "${var.network_name}-subnet-03"
 }
 
+# [START vpc_custom_create]
 module "test-vpc-module" {
-  source       = "../../"
+  source       = "terraform-google-modules/terraform-google-network/google"
+  version = "~> 3.2.0"
   project_id   = var.project_id
   network_name = var.network_name
+  mtu = var.mtu
```

Variables need to be declared in variables.tf to be used. In this case, we should just hard code a value.

```suggestion
  mtu = 1460
```